files/sinonjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "sinon",
+  "repo": "sinonjs/sinon",
+  "files": {
+    "basePath": "lib/",
+    "include": ["sinon.js"]
```

Comment:
The bot is using Github tags. For example https://github.com/sinonjs/sinon/tree/v2.0.0-pre
If they add their files there in all new versions then it will work.